pkg/quota/v1/evaluator/core/persistent_volume_claims.go

```diff
@@ -169,6 +197,11 @@ func (p *pvcEvaluator) Usage(item runtime.Object) (corev1.ResourceList, error) {
 		}
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(k8sfeatures.VolumeAttributesClass) && pvc.Spec.VolumeAttributesClassName != nil {
```

Comment:
Cab the VolumeAttributesClassName be an empty string in this step or it is validated before this.

Reply:
done in [7fd84542c](https://github.com/kubernetes/kubernetes/compare/4450eaa33bb22d2d8bfe8c0a7084306e82857918..7fd84542c847abf1482de170a570611a4faf61ae)